src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -116,6 +116,27 @@ class AppsResourceTest extends MarathonSpec with MarathonActorSupport with Match
     response.getMetadata.containsKey(RestResource.DeploymentHeader) should be(true)
   }
 
+  test("Do partial update with patch methods") {
```

Comment:
can we also have a test for PATCH where an app is rejected because it doesn't exist?